roles/fs-drift/templates/workload_job.yml.j2

```diff
@@ -42,29 +42,28 @@ spec:
           - name: uuid
             value: "{{ uuid }}"
           - name: test_user
-            value: "{{ test_user }}"
+            value: "{{ test_user | default("ripsaw") }}"
           - name: clustername
             value: "{{ clustername }}"
-{% if elasticsearch.server is defined %}
           - name: es
-            value: "{{ elasticsearch.server }}"
+            value: "{{ elasticsearch.server | default() }}"
```

Comment:
this doesn't make sense, if elasticsearch.server is defined, then why would you need to default it?   I think you meant "if elasticsearch is defined?

Reply:
@rsevilla87 what about this one?   This is my last comment - the one below is just another case of it.

Reply:
I've made several updates around this, take a look when you get time.